allennlp/semparse/domain_languages/wikitables_language.py

```diff
@@ -320,6 +320,21 @@ def get_agenda(self,
                 agenda.append(f"Number -> {number}")
         return agenda
 
+    @staticmethod
+    def is_instance_specific_entity(entity_name: str) -> bool:
```

Comment:
Hmm, seems like we should have a function for this on the base class, but maybe it doesn't fit...  I'll think about this some more.